ui/appui-react/src/test/widget-panels/Frontstage.test.tsx

```diff
@@ -1580,61 +1592,62 @@ describe("Frontstage local storage wrapper", () => {
 
         it("should hide the widget", () => {
           let nineZone = createNineZoneState();
-          nineZone = addPanelWidget(nineZone, "left", "w1", ["t1", "t2"]);
           nineZone = addTab(nineZone, "t1");
+          nineZone = addTab(nineZone, "t2");
+          nineZone = addPanelWidget(nineZone, "left", "w1", ["t1", "t2"]);
           const sut = setWidgetState(nineZone, new WidgetDef({ id: "t1" }), WidgetState.Hidden);
           sut.widgets.w1.tabs.should.eql(["t2"]);
         });
 
         it("should reopen hidden widget", () => {
           let nineZone = createNineZoneState();
-          nineZone = addFloatingWidget(nineZone, "w1", ["w1"]);
           nineZone = addTab(nineZone, "w1");
+          nineZone = addFloatingWidget(nineZone, "w1", ["w1"]);
           const widgetDef = new WidgetDef({ id: "w1", hideWithUiWhenFloating: true });
           let hideWidgetState = setWidgetState(nineZone, widgetDef, WidgetState.Hidden);
-          expect (hideWidgetState.floatingWidgets.byId.w1.hidden).to.be.true;
+          should().not.exist(hideWidgetState.floatingWidgets.byId.w1);
           let showWidgetState = setWidgetState(hideWidgetState, widgetDef, WidgetState.Open);
-          expect (showWidgetState.floatingWidgets.byId.w1.hidden).to.be.false;
+          expect(showWidgetState.floatingWidgets.byId.w1.hidden).to.be.false;
```

Comment:
I've seen this in a couple of places, why use `should()` and `expect()` ? (From what I've seen, `should()` https://www.chaijs.com/guide/styles/#should should be called to add the `.should.xxx` to all objects prototype, while it may work like we use it here, it seems to be potentially doing more than needed (althouth they probably check if the prototype is already patched...)

To me, it feels like we should stick to expect as it allows keeping the same structure even when testing for a non existing element (https://www.chaijs.com/guide/styles/#should-extras)
```
expect(hideWidgetState.floatingWidgets.byId.w1).to.not.exist;
```

Reply:
Removed `should()` assertions from Frontstage tests.